src/markdown-pages/docs/first-steps/installing-offline.md

```diff
@@ -31,7 +31,7 @@ Galasa tests and Managers are written in Java - you will need to install a Java
 
 If you do not already have an Eclipse installation, you can <a href="https://www.eclipse.org/downloads/packages/installer" target="_blank">download</a> a version of Eclipse appropriate for your machine. Choose a package that supports your required level of Java development - _Eclipse IDE for Java Developers_ or _Eclipse IDE for Java EE Developers_. If you are unsure, then the _Eclipse IDE for Java Developers_ should be fine, and you can always add plug-ins if and when you discover you need them.
 
-If you already have a version of Eclipse installed, it should be at the version codenamed Photon (released in June 2018) or later.  
+If you already have a version of Eclipse installed, it should be at the version codenamed Photon (released in June 2018) or later. _Note:_ The Java 17 version of the JustJ installation that is shipped as part of Eclipse causes Galasa test runs to fail. We recommend using a Java 8 or 11 JDK whilst we investigate this issue.
```

Comment:
Although the issue was reported with JustJ 17 it would probably affect any JDK at 17, we should mention that

we do also test on Java 16 so that should be ok. 

Apart from that this looks fine